src/components/AuthSecretKey.tsx

```diff
@@ -0,0 +1,131 @@
+import React, { useState } from "react";
+import { Link, useHistory } from "react-router-dom";
+import { useDispatch } from "react-redux";
+import styled from "styled-components";
+import { Keypair } from "stellar-sdk";
+
+import { fetchAccount } from "ducks/account";
+
+const Warning = styled.div`
+  background-color: #f3e5e5;
+  color: #681e1e;
+  padding: 20px;
+  margin-bottom: 20px;
+`;
+
+const TempLink = styled(Link)`
+  display: block;
+  margin-bottom: 20px;
+`;
+
+const TempButton = styled.button`
+  margin-bottom: 20px;
+`;
+
+const TempInput = styled.input`
+  margin-bottom: 20px;
+  min-width: 300px;
+`;
+
+export const AuthSecretKey = () => {
```

Comment:
I like it when my views' names fully explain what they do. This I think is a little ambiguous. I'd suggest something like "SecretKeyEntry" or "SecretKeyForm" or "Signin" or something.